src/ray/common/status.cc

```diff
@@ -102,6 +106,9 @@ std::string Status::CodeAsString() const {
       {StatusCode::ObjectAlreadySealed, STATUS_CODE_OBJECT_STORE_ALREADY_SEALED},
       {StatusCode::ObjectStoreFull, STATUS_CODE_OBJECT_STORE_FULL},
       {StatusCode::TransientObjectStoreFull, STATUS_CODE_TRANSIENT_OBJECT_STORE_FULL},
+      {StatusCode::JobNotStarted, STATUS_CODE_JOB_NOT_STARTED},
+      {StatusCode::WorkerPendingRegistration, STATUS_CODE_WORKER_PENDING_REGISTRATION},
+      {StatusCode::RuntimeEnvCreationFailed, STATUS_CODE_RUNTIME_ENV_CREATION_FAILED},
```

Comment:
It looks like we don't catch all these status inside a caller and do special action. Why don't we just use one status and write better error message inside? (e.g., WorkerPoolStartupNotReady Status?)

Reply:
Code updated. I have added some different branches by status.